src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+.page {
+  font-family: $main-font-family;
```

Comment:
The variable `$main-font-family` is used for the font family. Ensure that this variable is defined elsewhere in your SCSS files, otherwise it will cause a compilation error.